changelog.md

```diff
@@ -2,9 +2,19 @@
 
 All notable changes to this project will be documented in this file.
 
-The format is based on [Keep a Changelog](http://keepachangelog.com/) 
+The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
+## 3.0.0 - Upgraded `esling-config-airbnb peer dependencies`
+### Changed
```

Comment:
We need to dramatically improve the changelog here i think. I know you copied what I had put in #4, but i'll admit that it was rubbish! Essentially we need to itemise every major difference here (think of the things we had to fix in the backpack monorepo upgrade).

Reply:
I agree, but my concern is that there are so many changes, new rules etc that we'll end up duplicating their changelog inside our own repo. I think it would be clearer if we just provided links to the official changelogs, e.g.:

```
### Changed
- Upgraded the following peer dependencies:
 - babel-eslint:           `^7.2.3` -> `^8.0.1`
 - [eslint](https://github.com/eslint/eslint/blob/master/CHANGELOG.md): `^3.17.1` -> `^4.9.0`
 - [eslint-config-airbnb](https://github.com/airbnb/javascript/blob/master/packages/eslint-config-airbnb/CHANGELOG.md): `^14.1.0` -> `^16.1.0`
 - [eslint-plugin-import](https://github.com/benmosher/eslint-plugin-import/blob/master/CHANGELOG.md): `^2.2.0`  -> `^2.8.0`
 - [eslint-plugin-jsx-a11y](https://github.com/evcohen/eslint-plugin-jsx-a11y/blob/master/CHANGELOG.md): `^4.0.0`  -> `^6.0.2`
 - [eslint-plugin-react](https://github.com/yannickcr/eslint-plugin-react/blob/master/CHANGELOG.md): `^6.10.0` -> `^7.4.0`

There are some substantial rule changes in this upgrade so you may find that your linter fails after upgrading. [eslint --fix](https://eslint.org/docs/user-guide/command-line-interface#--fix) should be able to fix the majority of these automatically. Anything else will need to be fixed manually.
```

What do you think?